docs/langref/src/states.md

```diff
@@ -33,5 +33,5 @@ export component Example inherits Window {
 ```
 
 In this example, the `active` and `active-hovered` states are defined depending on the value of the `active`
-boolean property and the `TouchArea`'s `has-hover`. So hovering will toggle between blue and green background,
+boolean property and the `TouchArea`'s `has-hover`. Hovering will toggle between blue and green background,
```

Comment:
```suggestion
boolean property and the `TouchArea`'s `has-hover`. When the user hovers the example with the mouse, it will toggle between a blue and a green background,
```